src/pgduckdb_guc.cpp

```diff
@@ -225,4 +234,72 @@ InitGUC() {
 	                           &duckdb_disabled_filesystems, PGC_SUSET);
 }
 
+extern "C" {
+
+#if PG_VERSION_NUM < 170000
+
+struct config_generic *
+get_config_handle(const char *name) {
+	struct config_generic *gen = nullptr;
+#if PG_VERSION_NUM >= 160000
+	int var_count = 0;
+	auto *guc_variables = get_guc_variables(&var_count);
```

Comment:
Can you use this for loop only for versions below 16? And use `find_option` for 16 and up?